test/Integration/conv-to-matmul.mlir

```diff
@@ -1,21 +1,6 @@
-// RUN: tpp-opt %s -bufferize -convert-linalg-to-loops -convert-vector-to-scf -convert-scf-to-cf -expand-strided-metadata -lower-affine -convert-arith-to-llvm -convert-vector-to-llvm -finalize-memref-to-llvm -arith-expand -convert-math-to-llvm -convert-func-to-llvm -reconcile-unrealized-casts | \
-// RUN: mlir-cpu-runner \
-// RUN:  -e entry -entry-point-result=void  \
-// RUN: -shared-libs=%llvmlibdir/libmlir_c_runner_utils%shlibext | \
-// RUN: FileCheck %s
-//
-
+// This should really be in the passes directory, not here
```

Comment:
I think in general it's fine to have some small IR sanity checks in these tests.
Whether these already present are strictly necessary is another question.
Any more insights from your side?

Reply:
Nope, exactly what you said. Just a comment for the future.